recipes/alphalens/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "alphalens" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "71e8136d3f5142609905a905d25b349c9b8d50f1df327fe6921398d84fc96a0b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - ipython >=3.2.3
+    - matplotlib >=1.4.0
+    - numpy >=1.9.1
+    - pandas >=0.18.0
+    - scipy >=0.14.0
+    - seaborn >=0.6.0
+    - statsmodels >=0.6.1
+
+test:
+  imports:
+    - alphalens
+    - alphalens.tests
+
+about:
+  home: http://quantopian.github.io/alphalens/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: 'Performance analysis of predictive (alpha) stock factors'
+
+  description: |
+    Alphalens is a Python Library for performance analysis of predictive (alpha) stock factors. Alphalens works great with the Zipline open source backtesting library, and Pyfolio which provides performance and risk analysis of financial portfolios.
```

Comment:
Could you please break this up over multiple lines?

Reply:
Yes sure!